### PR TITLE
feat: clone demo modelinstancess and testinstances when a new user si…

### PR DESCRIPTION
**Pull request by zsinnema**
#469 default test and models for new users
https://github.com/MetaCell/geppetto-scidash/issues/469

## Changes
* clone model and test instance on new user sign up
* models files reformated to PEP8

## Remarks:
* added post install step to README.md

    ## Post install steps
   For copying/cloning initial model instances and test instances please update the 
   DEMO_USER_ID in the settings file to point to the user (id) from where the models and test 
   instances should be cloned.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -21,3 +21,5 @@ GEPPETTO_SERVLET_URL=ws://${GEPPETTO_HOST}:${GEPPETTO_PORT}/org.geppetto.fronten
 GEPPETTO_BASE_URL=http://${GEPPETTO_HOST}:${GEPPETTO_PORT}/org.geppetto.frontend/geppetto
 
 BASE_PROJECT_FILES_HOST='http://172.21.0.1:8000/static/projects/'
+
+DEBUG=True
```
Comment (ddelpiano) [inline .env:25]: @zsinnema do you want to leave this in intentionally or should we remove it from the pr?

File: README.md [hunk 2]
```diff
@@ -327,3 +327,9 @@ Also codefresh repository requires especial secret for pulling images:
       imagePullSecrets:
       - name: codefresh-generated-r.cfcr.io-cfcr-scidash-testing
 ```
+
+## Post install steps
+
+For copying/cloning initial model instances and test instances please update the DEMO_USER_ID in the settings file
```
Comment (ddelpiano) [inline README.md:333]: Question, is this copying all the tests and models that belong to this user? Thx

**Review by ddelpiano — COMMENTED** [review]
waiting @zsinnema to confirm on the points commented and this is good to go after few tweaks either in the db or in the settings to take by default the admin user.

File: README.md (hunk 2 above)
Comment (zsinnema) [reply README.md:333]: Yes, but with the "new scidash" it is even better implemented in settings.py
`DEBUG = TEMPLATE_DEBUG = os.environ.get('DEVELOPMENT', '0') == '1'`

the .env file is copied on install from service/dotenv and there I didn't put it.
So when a developer checks out the repo the DEBUG is True

**Pull request merged**